cmd/infra/powervs/destroy.go

```diff
@@ -492,8 +492,20 @@ func destroyPowerVsCloudConnection(ctx context.Context, options *DestroyInfraOpt
 			}
 		}
 	} else {
+
+		deleteCC := func(id string) error {
+			for i := 0; i < 5; i++ {
+				err = deletePowerVsCloudConnection(options, id, client, jobClient)
+				if err == nil {
```

Comment:
nit: the if err can be inlined

Reply:
Also by reading the ticket I'd expect this error to be checked against the known cases which require retry

Reply:
Inlined the err.
Want to retry the delete operation in all the cases, since missing to delete cloud connection causes it to become a stale resource. We feel there is no harm in retrying the delete operation if it failed.